src/main/include/hardware/HardwareInterface.h

```diff
@@ -0,0 +1,20 @@
+
+#pragma once
+
+#include <frc/drive/DifferentialDrive.h>
+
+#include "hardware/ActuatorInterface.h"
+#include "hardware/SensorInterface.h"
+
+typedef struct hardware_t
```

Comment:
this I can't even pretend to know what's going on

Reply:
This is defining our centralized hardware management struct